### PR TITLE
Depend on OL 3.5 and Cesium 1.9

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by gberaudo** [comment]
Thanks for any review.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 dist
 node_modules
 examples/example-list.js
+*.swp
```
Comment (ahocevar) [inline .gitignore]: Shouldn't this be in your local gitignore file?

Comment (ahocevar) [inline .gitignore]: Or are the `*.swp` files really generated by the build process?

Comment (gberaudo) [inline .gitignore]: OK, I removed this line and instead created a global gitignore file on my machine like explained in https://help.github.com/articles/ignoring-files/.

**Comment by ahocevar** [comment]
Thanks! Please merge.

**Comment by gberaudo** [comment]
Thank you for the review @ahocevar.

**Pull request merged**